samples/stocks/README.md

```diff
@@ -0,0 +1,11 @@
+# /fusebot stocks <ticker>
```

Comment:
The `<ticker>` part doesn't show up. You probably need to use some HTML entities like `&lt;`, `&gt;` or maybe some special Markdown stuff.

See: https://github.com/fusebit/fusebot/blob/ef8c5b22c9daa96fe181bcc2fcad315a53833dac/samples/stocks/README.md

Reply:
good catch, fixed